### ISSUE TITLE
dashboard does not work on HTTPS

### ISSUE THREAD

**Issue by turtleDeng** [emqx/emqx#3249]
#### Environment

- OS: MacOSX
- Erlang/OTP: 21.3
- EMQ: 4.0.2

#### Description

```
##--------------------------------------------------------------------
## HTTPS Listener

dashboard.listener.https = 18084
dashboard.listener.https.acceptors = 2
dashboard.listener.https.max_clients = 512
dashboard.listener.https.keyfile = etc/certs/key.pem
dashboard.listener.https.certfile = etc/certs/cert.pem
dashboard.listener.https.cacertfile = etc/certs/cacert.pem
dashboard.listener.https.tls_versions = tlsv1.2,tlsv1.1,tlsv1
dashboard.listener.https.ciphers = ECDHE-ECDSA-AES256-GCM-SHA384,ECDHE-RSA-AES256-GCM-SHA384,ECDHE-ECDSA-AES256-SHA384,ECDHE-RSA-AES256-SHA384,ECDHE-ECDSA-DES-CBC3-SHA,ECDH-ECDSA-AES256-GCM-SHA384,ECDH-RSA-AES256-GCM-SHA384,ECDH-ECDSA-AES256-SHA384,ECDH-RSA-AES256-SHA384,DHE-DSS-AES256-GCM-SHA384,DHE-DSS-AES256-SHA256,AES256-GCM-SHA384,AES256-SHA256,ECDHE-ECDSA-AES128-GCM-SHA256,ECDHE-RSA-AES128-GCM-SHA256,ECDHE-ECDSA-AES128-SHA256,ECDHE-RSA-AES128-SHA256,ECDH-ECDSA-AES128-GCM-SHA256,ECDH-RSA-AES128-GCM-SHA256,ECDH-ECDSA-AES128-SHA256,ECDH-RSA-AES128-SHA256,DHE-DSS-AES128-GCM-SHA256,DHE-DSS-AES128-SHA256,AES128-GCM-SHA256,AES128-SHA256,ECDHE-ECDSA-AES256-SHA,ECDHE-RSA-AES256-SHA,DHE-DSS-AES256-SHA,ECDH-ECDSA-AES256-SHA,ECDH-RSA-AES256-SHA,AES256-SHA,ECDHE-ECDSA-AES128-SHA,ECDHE-RSA-AES128-SHA,DHE-DSS-AES128-SHA,ECDH-ECDSA-AES128-SHA,ECDH-RSA-AES128-SHA,AES128-SHA
```

**Comment by HJianBo** [comment]
The dashboard https listener is ok. But the management https listener misses some params.

I have committed it:
https://github.com/emqx/emqx-management/pull/172/files

**Comment by HJianBo** [comment]
It has fixed at v4.0.3
